lighthouse-core/gather/gather-runner.js

```diff
@@ -494,6 +494,39 @@ class GatherRunner {
     };
   }
 
+  /**
+   * Populates the important base artifacts from a fully loaded test page.
+   * Currently must be run before `start-url` gatherer so that `WebAppManifest`
+   * will be available to it.
+   * @param {LH.Gatherer.PassContext} passContext
+   * @return {LH.Artifacts.} passContext
+   */
+  static async getInstallabilityErrors(passContext) {
```

Comment:
gather runner tests for this method would be nice :D

Reply:
done